requirements/requirements.in

```diff
@@ -44,8 +44,8 @@ Unidecode==0.04.20
 xlrd==1.0.0
 simplejson~=3.11
 sh==1.09
-gevent==1.2.2
-greenlet==0.4.12
+gevent==1.4.0
```

Comment:
the previous gevent upgrade caused issues on prod https://github.com/dimagi/commcare-hq/pull/22237 though it may have been related to older celery

Reply:
I haven't tested it.  It's in this PR because the current version blocks installing requirements.